### DISCUSSION TITLE
[Support]: Convert H265 to H264

### DISCUSSION THREAD

**Discussion by xttlegendapi** [blakeblackshear/frigate#12000] [General Support]
### Describe the problem you are having

My camera only supports H265, but that does not work with some devices. Is there any way to convert the rtsp stream to H264 or is this very resource intesive

### Version

-

### Frigate config file

```yaml
-
```

### Relevant log output

```shell
-
```

### FFprobe output from your camera

```shell
-
```

### Operating system

Debian

### Install method

Docker Compose

### Object Detector

CPU (no coral)

### Network connection

Wired

### Camera make and model

-

**Comment by NickM-27** [comment]
It is resource intensive but in any case, you'd use go2rtc to do this
